packages/rx_bloc_cli/example/README.md

```diff
@@ -571,4 +571,4 @@ For more info please visit [widget_toolkit_pin](https://pub.dev/packages/widget_
 [patrol_cli_pub_lnk]: https://pub.dev/packages/patrol_cli
 [patrol_native_integration_lnk]: https://patrol.leancode.pl/getting-started#integrate-with-native-side
 [rx_bloc_cli_cd_setup_lnk]: docs/continuous_delivery.md
-[golden_tests_with_smart_widgets_lnk]: https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/doc/golden_tests.md
+[golden_tests_with_smart_widgets_lnk]: docs/golden_tests.md
```

Comment:
I also didn't quite understand this from the discussion this morning. We should prefer absolute paths wherever possible, except the pages shown on `pub.dev`?

Reply:
Now that the files are being generated (part of the generated project), we can refer to them using relative paths. This should work with the example directory as well.

But to answer your question: it's the other way around. Relative paths won't work on pub.dev as we expect, so that's why they should contain a full URL.

Reply:
@DDavidPrime 
The URLs must work on pub.dev. Please fix this before you start with the release process.